nondex-test/src/test/java/edu/illinois/nondex/core/HashMapTest.java

```diff
@@ -148,6 +148,17 @@ public void testEntrySet() {
                 "{0=0, 1=1, 2=2, 3=3, 4=4, 5=5, 6=6, 7=7, 8=8, 9=9}", entrySet.toString());
     }
 
+    @Test
```

Comment:
Please use `@Test(expected=....)` to simplify the test when it should pass when an exception is thrown.

Reply:
Will incorporate these changes